### DISCUSSION TITLE
Is there any clean and fully equipped example html + css + js?

### DISCUSSION THREAD

**Discussion by Stakdek** [GrapesJS/grapesjs#3858] [Q&A]
Your documentation is good. But your “final result” just shows how it looks, and not the full code of how this happened. In the chapters before, you are “kinda” describing how to set up the Editor, but with all code snippets mixed together, I did not have any usable results…

Can you please provide a CLEAN and Fully equipped Editor JS/CSS and Sources to use?
Your demo (https://grapesjs.com/demo.html) is nice, and I would like to use this shown editor with its configuration. But the demo.html has too much overhead to filter and cannot be used properly…

Please help

I am working on an integration with Zope…

**Comment by artf** [comment]
The code of the final example is here in docs https://github.com/artf/grapesjs/blob/dev/docs/.vuepress/components/DemoTheme.vue
